aptos-move/af-cli/src/main.rs

```diff
@@ -28,12 +28,12 @@ pub enum AfCommands {
 fn main() -> Result<()> {
     let error_descriptions: ErrorMapping = bcs::from_bytes(cached_framework_packages::error_map())?;
     let args = AfCli::parse();
-    match &args.cmd {
```

Comment:
@gregnazario  do we actually still use this CLI? I thought that's all done via the `aptos` CLI now?

Reply:
Yes, there are some features that aren't in the aptos CLI but we should put in

Reply:
Lets keep this until we are sure we covered all features in the aptos cli.